code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -46,4 +46,6 @@
 	var/gender_ambiguous = 0 //if something goes wrong during gender reassignment this generates a line in examine
 
 	var/blood_max = 0 //how much are we bleeding
-	var/bleedsuppress = 0 //for stopping bloodloss, eventually this will be limb-based like bleeding
\ No newline at end of file
+	var/bleedsuppress = 0 //for stopping bloodloss, eventually this will be limb-based like bleeding
+
+	var/datum/martial_art/martial_art = null
```

Comment:
I was considering this but then again animals are not known for varied fighting and many critters already have unique attack_whatever defined so subtypes overriding this seemed enough. I can see how much i'd need to move around and try doing this later today maybe.